src/Simulation/QCTraceSimulator/QubitMetrics.cs

```diff
@@ -10,24 +10,14 @@ namespace Microsoft.Quantum.Simulation.QCTraceSimulatorRuntime
     /// </summary>
     public class QubitTimeMetrics
```

Comment:
Is this class still needed? It seems only the depth counter is using it, so it may make sense to absorb it into the depth counter rather than exposing it as part of the public API, especially since it now only contains the qubit Id.

Reply:
I would definitely like to get rid of this class entirely, but I cannot because of IQCTraceSimulatorListener. This is a public interface used by many listeners including depth counter. It has "OnPrimitiveOperation" method, and qubits or qubit ids are not passed to this method, only attached metrics. I need ids in that method that's why I put them into metrics.
The new approach to tracer has this changed so it is easier to add qubit tracking there.